com.civicrm.hrjobroles/js/test/controllers/hr-job-roles-controller/hr-job-roles-controller_test.js

```diff
@@ -27,6 +27,91 @@ define([
             DateValidation.dateFormats.push('DD/MM/YYYY');
         }));
 
+        describe('getOptionValues', function() {
+
+          it('should build the "CostCentreList" array of objects containing the "weight" property', function(){
```

Comment:
Small thing: the `it` statements should be in a declarative form, so instead of `it('should do X')`, it should be `it('does X')`